scripts/seedDb.mjs

```diff
@@ -51,3 +56,25 @@ async function recreateLeagueData(leagueKeyPrefix, dataRepo) {
         await redis.json.set(leagueKeyPrefix+user.userId, "$", userString)
     }
 }
+
+// Create league configuration data
+
+await recreateLeagueConfigurationData("league_configuration:amazing_race:35:", amazingRace35LeagueConfiguration)
+await recreateLeagueConfigurationData("league_configuration:amazing_race:36:", amazingRace36LeagueConfiguration)
+await recreateLeagueConfigurationData("league_configuration:big_brother:26:", bigBrother26LeagueConfiguration)
+await recreateLeagueConfigurationData("league_configuration:survivor:47:", survivor47LeagueConfiguration)
+
+async function recreateLeagueConfigurationData(leagueKeyPrefix, dataRepo) {
+
+    const leagueConfigurationCursor = await redis.scan("0", {match: leagueKeyPrefix+"*"})
+
+    for (const aKey of leagueConfigurationCursor[1]) { //list of keys
+        redis.del(aKey)
+    }
+
+    const leagueConfigString = JSON.stringify(dataRepo)
```

Comment:
Also... I get inserting these as is... but I am also wondering if we shouldn't just do a quick stop gap to change them a bit first so that we are inserting them the way we want the to be saved into the DB. 🤔

Reply:
I'd love to talk this out with you because I want to hear what you recommend

Reply:
Update: Did some clean up in this PR
https://github.com/jjm3x3/AmazingRaceFantasy/pull/222